tests/conftest.py

```diff
@@ -352,11 +352,11 @@ class EmptyTool(OpenAITool):
 def fixture_mirascope_user_settings() -> MirascopeSettings:
     """Returns a `MirascopeSettings` instance."""
     return MirascopeSettings(
-        format_command="ruff format",
-        mirascope_location=".test",
+        format_command="ruff check --select I --fix; ruff format",
```

Comment:
should we update this to be the default command in the generic mirascope.ini?

Reply:
it already is